http-add-on/templates/deployment.yaml

```
@@ -0,0 +1,67 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    control-plane: controller-manager
+    httpscaledobjects.http.keda.sh/version: {{ .Values.version }}
+    keda.sh/addon: {{ .Chart.Name }}
+    app: {{ .Chart.Name }}
+    name: {{ .Chart.Name }}-controller-manager
+    app.kubernetes.io/name: {{ .Chart.Name }}-controller-manager
+    app.kubernetes.io/version: {{ .Values.version }}
+    app.kubernetes.io/component: controller-manager
+    app.kubernetes.io/part-of: {{ .Chart.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+{{ tpl .Values.additionalLabels . | indent 4}}
+  name: {{ .Chart.Name }}-controller-manager
+  namespace: {{ .Release.Namespace }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      control-plane: controller-manager
+      httpscaledobjects.http.keda.sh/version: {{ .Values.version }}
+      keda.sh/addon: {{ .Chart.Name }}
+  template:
+    metadata:
+      labels:
+        control-plane: controller-manager
+        httpscaledobjects.http.keda.sh/version: {{ .Values.version }}
+        keda.sh/addon: {{ .Chart.Name }}
+        app: {{ .Chart.Name }}
+        name: {{ .Chart.Name }}-operator
+        app.kubernetes.io/name: {{ .Chart.Name }}-operator
+        app.kubernetes.io/version: {{ .Values.version }}
+        app.kubernetes.io/component: operator
+        app.kubernetes.io/part-of: {{ .Chart.Name }}
+        app.kubernetes.io/managed-by: {{ .Release.Service }}
+        app.kubernetes.io/instance: {{ .Release.Name }}
+    {{ tpl .Values.additionalLabels . | indent 4}}
+    spec:
+      containers:
+      - args:
+        - --secure-listen-address=0.0.0.0:{{ .Values.operator.port | default 8443 }}
+        - --upstream=http://127.0.0.1:8080/
+        - --logtostderr=true
+        - --v=10
+        image: gcr.io/kubebuilder/kube-rbac-proxy:v0.5.0
+        name: kube-rbac-proxy
+      - args:
+        - --metrics-addr=127.0.0.1:8080
+        - --enable-leader-election
+        image: "{{ .Values.images.operator }}:{{ .Values.version | default .Chart.AppVersion }}"
+        imagePullPolicy: {{ .Values.operator.pullPolicy | default "Always" }}
+        name: {{ .Chart.Name }}-operator
+        env:
+        - name: KEDAHTTP_OPERATOR_EXTERNAL_SCALER_IMAGE
+          value: "{{ .Values.images.scaler }}:{{ .Values.version | default .Chart.AppVersion }}"
```

Apparently our tags have a `v` in it
```suggestion
          value: "{{ .Values.images.scaler }}:v{{ .Values.version | default .Chart.AppVersion }}"
```